src/components/components/FeaturedJobCard/FeaturedJobCard.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import "./FeaturedJobCard.scss";
+
+function FeaturedJobCard({ job, logoNameToPath }) {
+  return (
+      <div className="FeaturedJobCard">
+        <div className="FeaturedJobCard__header">
+          <img
+            className="FeaturedJobCard__header__img"
+            src={logoNameToPath}
+            alt="Company Logo"
```

Comment:
As an alt-Text you can use job.logoName or job.company

Reply:
done!